README.md

```diff
@@ -13,31 +13,67 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 -->
+
+<img align="left" width="150" src="assets/images/logo.png" alt="adios logo" /><br />
+
 # Adios: One-stop solution for Google Ads image assets management
```

Comment:
Perhaps we can remove "Adios:" since we have the logo right above.

```suggestion
# A one-stop solution for Google Ads image assets management
```

Reply:
From the SEO POV I would leave the name since it is in the H1 tag